src/components/button/index.js

```diff
@@ -57,6 +58,7 @@ Button.propTypes = {
   onClick: PropTypes.func,
   className: PropTypes.string,
   active: PropTypes.bool,
+  type: PropTypes.string,
```

Comment:
If it can be only one of "submi", "reset" or "button" I would use the `oneOf` prop type here.

Reply:
Please change it before merging. I would squash it with the previous commit ;)